### ISSUE TITLE
Support for mrxs files (3dhisttech)?

### ISSUE THREAD

**Issue by markemus** [ome/bioformats#3657]
Are there any plans to add support for the mrxs format?

https://openslide.org/formats/mirax/

Currently this format is detected as JPEG by `bfconvert` and only the thumbnail is extracted.

**Comment by joshmoore** [comment]
Hi @markemus,

*tl;dr* There's no plan for Bio-Formats to support 3D HISTECH .mrxs files. Please see https://forum.image.sc/t/converting-whole-slide-images-to-ome-tiff-a-new-workflow/32110 for information on how to convert your data.

If you would like to know the reasoning behind that decision, please see https://blog.openmicroscopy.org/file-formats/community/2016/01/06/format-support/ which (in excerpted form) stated:

> 3D HISTECH .mrxs is an example of a complex format, the design of which does not make our work any easier. In fact, we can say with some confidence that the 3D HISTECH .mrxs file format is the most complex whole slide imaging file format we have ever encountered.
> ...
> The OME Consortium and the wider development community have worked steadily since 2002, funded mostly by grants from non-profit charities and public funders, to build tools for the scientific community.
>
> Building readers for proprietary formats has never been funded, and we don’t think it would ever be funded by any grant funding panel....
>
> Perhaps we could crowdsource the funding for file formats but that misses the point—the formats we often lack the resources to support are those which are complex, expensive, difficult, proprietary, closed formats, designed to lock their users into a single, proprietary software application. The community’s resources are finite and should be used for things other than reverse engineering this type of format; work that, if subjected to peer review, would be declined as a waste of community resources.

The current solution from the OME community is to convert your mrxs files into an open format. We offer a pipeline for doing so to OME-TIFF which you can read about on the [Glencoe blog](https://www.glencoesoftware.com/blog/2019/12/09/converting-whole-slide-images-to-OME-TIFF.html) ([forum](https://forum.image.sc/t/converting-whole-slide-images-to-ome-tiff-a-new-workflow/32110)). More details about the longer term plan for file formats are on the [OME blog](https://blog.openmicroscopy.org/community/file-formats/2019/06/25/formats/) ([forum](https://forum.image.sc/t/ome-s-position-regarding-file-formats/26952)).





All the best,
~Josh

P.S. there's been more discussion around this topic on the forums that you might find interesting: https://forum.image.sc/search?q=mrxs%20bio-formats
